makefile

```diff
@@ -1,19 +1,80 @@
-format:
-	poetry run python -m pycln --all .
-	poetry run python -m isort format .
-	poetry run python -m black .
+HELPERS_DIR := tests/helpers
+NOTEBOOKS_DIR := notebooks
+CONVERTED_NOTEBOOKS_TEMP_DIR := tests/_temp/notebooks
+POETRY_RUN_PYTHON := poetry run python
```

Comment:
nit/low confidence: don't love this variable because makefiles don't have hover definitions so you have to scroll around for it, and it's as long as the actual command

Reply:
They're all short, but I moved them into variables because they are reused multiple times